packages/react-components/src/templates/PageConstraints.tsx

```diff
@@ -0,0 +1,71 @@
+import { css, SerializedStyles } from '@emotion/react';
+import { ReactNode } from 'react';
+
+import { rem, smallDesktopScreen } from '../pixels';
+
+const containerStyles = css({
+  display: 'flex',
+  flexFlow: 'column',
+  padding: `${rem(48)} ${rem(88)}`,
+  [`@media (max-width: ${smallDesktopScreen.width - 1}px)`]: {
+    padding: `${rem(48)} ${rem(24)}`,
+  },
+});
+
+const constrainedContainerStyles = css({
+  maxWidth: rem(1600),
+  width: '100%',
+  margin: '0 auto',
+});
+
+const noPaddingBottomStyles = css({
+  paddingBottom: 0,
+  [`@media (max-width: ${smallDesktopScreen.width - 1}px)`]: {
+    paddingBottom: 0,
+  },
+});
+
+const noPaddingTopStyles = css({
+  paddingTop: 0,
+  [`@media (max-width: ${smallDesktopScreen.width - 1}px)`]: {
+    paddingTop: 0,
+  },
+});
+
+type PageConstraintsProps = {
+  children: ReactNode;
+  as?: 'header' | 'main' | 'div' | 'article';
+  noPaddingBottom?: boolean;
+  noPaddingTop?: boolean;
+  unconstrainedStyles?: SerializedStyles;
+};
+
+/**
+ * A layout component that provides consistent page-width constraints and horizontal
+ * padding across different screen sizes.
+ *
+ * Use `PageConstraints` to create consistent layout boundaries for page sections.
+ * It applies responsive horizontal padding and max-width constraints to ensure
+ * content sections behave uniformly across different screen sizes and
+ * maintains visual consistency throughout the application.
+ */
+const PageContraints: React.FC<PageConstraintsProps> = ({
```

Comment:
Can we fix this typo, please? 🙏🏻 
```suggestion
const PageConstraints: React.FC<PageConstraintsProps> = ({
```

Reply:
Oops! Embarrassingly obvious typo, will commit a massive renaming, thanks!